public/index.html

```diff
@@ -23,6 +23,8 @@
   </head>
   <body>
     <div id="react-root" />
+    <!-- TODO: Change from google dependancy to our Backend service -->
```

Comment:
I don't think this is possible, it is not documented: https://developers.google.com/recaptcha/docs/display

But we could dynamically load the recaptcha javascript so that it is not loaded on all the pages.

[no action required]